pkg/scheduler/framework/types.go

```diff
@@ -144,6 +144,13 @@ func (ce ClusterEvent) IsWildCard() bool {
 	return ce.Resource == WildCard && ce.ActionType == All
 }
 
+// Match returns true if ClusterEvent is matched with the coming event.
+// If the ce.Resource is "*", there's no requirement for the coming event' Resource.
+// Contrarily, if the coming event's Resource is "*", the ce.Resource should only be "*".
+func (ce ClusterEvent) Match(event ClusterEvent) bool {
+	return ce.IsWildCard() || (ce.Resource == WildCard || ce.Resource == event.Resource) && ce.ActionType&event.ActionType != 0
```

Comment:
Should we? (I know this suggestion wouldn't change anything in `isPodWorthRequeuing` though.)

```suggestion
	return ce.IsWildCard() || event.IsWildCard() || (ce.Resource == WildCard || ce.Resource == event.Resource) && ce.ActionType&event.ActionType != 0
```

Reply:
I haven't seen emitting a wildcard event yet, what does the coming event is a wildcard means? **We can definitely know what the coming event is**, so let's get rid of the " * " .
TL;DR
When the coming event is "*", it will only matched with `ce.Resource == event.Resource) && ce.ActionType&event.ActionType != 0`, make sense?

Reply:
```
		// If the wildcard event is special one as someone wants to force all Pods to move to activeQ/backoffQ.
		// We return queueAfterBackoff in this case, while resetting all blocked plugins.
```

Got the idea here, but I still take it as an unmatched event but we have a special case here. I'll add the comment.

Reply:
> I haven't seen emitting a wildcard event yet, what does the coming event is a wildcard means?

My comment at https://github.com/kubernetes/kubernetes/pull/123117#discussion_r1477207815 describes that -- the scheduling queue uses the wildcard event when moving Pods from unsched to backoffQ forcibly. ([`flushUnschedulablePodsLeftover`](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/internal/queue/scheduling_queue.go#L841))
Given this usecase, we've defined an incoming wildcard event to match all registered events and ignore QueueingHInts.

Reply:
Yes, I didn't know that before, something new learned.